packages/@aws-cdk/aws-lambda/lib/alias.ts

```diff
@@ -235,6 +236,14 @@ export class Alias extends QualifiedFunctionBase implements IAlias {
     });
   }
 
+  public createFunctionUrl(options: FunctionUrlOptions): FunctionUrl {
```

Comment:
All properties are optional, so passing the struct should be optional.

```suggestion
  public createFunctionUrl(options: FunctionUrlOptions = {}): FunctionUrl {
```